src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+              <PvCheckbox
+                v-model="specialParam"
+                input-id="eye-tracking"
+                value="hasEyeTracking"
+                @change="checkBoxStatus"
+              />
+              <label class="ml-2 mr-3 flex cursor-pointer" style="width: 80%" for="eye-tracking">Eye - Tracking</label>
+              <i class="pi pi-eye" style="font-size: 1rem; width: 20%"></i>
+            </div>
+          </div>
+        </div>
+        <div class="hidden">
+          <h3 class="mb-4 mt-4">Consent Amount and Expected Time</h3>
+          <div class="flex flex-row">
+            <div class="mr-1">
+              <span class="p-float-label">
+                <PvInputText id="consent-amount" v-model="amount" class="w-full" disabled="true" />
+                <label for="consent-amount" class="text-sm w-full">Payment Amount $$</label>
+              </span>
+            </div>
+            <div class="ml-3">
+              <span class="p-float-label">
+                <PvInputText id="consent-time" v-model="expectedTime" class="w-full" disabled="true" />
+                <label for="consent-time" class="text-sm w-full">Expected Time Amount</label>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-if="knowWhatIWant" class="flex flex-column pl-3" style="width: 50%">
+        <h3>Select a Consent Form</h3>
+        <PvDropdown
+          v-model="selectedConsent"
+          :options="listOfDocs.consent"
+          optionLabel="fileName"
+          style="width: 70%"
+          @change="updateConsent"
+        />
+        <h3 class="pt-3">Select an Assent Form</h3>
+        <PvDropdown
+          v-model="selectedAssent"
+          :options="listOfDocs.assent"
+          optionLabel="fileName"
+          style="width: 70%"
+          @change="updateAssent"
+        />
+        <div div class="hidden">
+          <h3 class="mb-4 mt-5">Consent Amount and Expected Time</h3>
+          <div class="flex flex-row">
+            <div class="mr-1">
+              <span class="p-float-label">
+                <PvInputText id="consent-amount" v-model="amount" class="w-full" disabled="true" />
+                <label for="consent-amount" class="text-sm w-full">Payment Amount $$</label>
+              </span>
+            </div>
+            <div class="ml-3">
+              <span class="p-float-label">
+                <PvInputText id="consent-time" v-model="expectedTime" class="w-full" disabled="true" />
+                <label for="consent-time" class="text-sm w-full">Expected Time Amount</label>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-if="knowWhatIWant || helpMeSelect" class="flex-column" style="width: 50%">
+        <h3 class="font-bold text-center text-xl">Suggested Forms</h3>
+        <div class="w-full">
+          <PvFieldset legend="Consent" v-if="consents && consents.length > 0">
```

Comment:
Love the outlined box for consent and assent.

Reply:
Thank you :)